src/Rules/Documentation.hs

```diff
@@ -34,6 +37,12 @@ buildPackageDocumentation context@Context {..} =
             -- HsColour sources
             -- TODO: what is the output of GhcCabalHsColour?
             whenM (isSpecified HsColour) $ do
+                top <- interpretInContext context getTopDirectory
+                let installDistDir = top -/- buildPath context
```

Comment:
What about doing install-related copying in install rules? The current solution seems to always do the copying, which is suboptimal.